act-rules-format.bs

```diff
@@ -245,183 +259,212 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
+### Expectations for Atomic Rules ### {#expectations-atomic}
 
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. When atomic rules are used in composite rules, the outcome of the composite rules can be `passed` even if the outcome of one of its constituent atomic rule is `failed`, depending on the [expectations](#expectations-composite) of the composite rule.
+An [=atomic rule=] expectation MUST only use information available in the [input aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures that atomic rules are encapsulated.
 
 <div class=example>
-  <p><strong>Example</strong>: A rule for labels of HTML `input` elements might have the following expectations:</p>
+  <p>**Example:** A rule for labels of HTML `input` elements might have the following expectations:</p>
   <ol>
-    <li>The test target has an accessible name (as described in [Accessible Name and Description: Computation and API Mappings 1.1](https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te)). [[accname-aam-1.1]]</li>
-    <li>The accessible name describes the purpose of the test target.</li>
+    <li>Each test target has an accessible name (as described in [Accessible Name and Description: Computation and API Mappings 1.1](https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te)). [[accname-aam-1.1]]</li>
+    <li>The accessible names describe the purpose of each test target.</li>
   </ol>
 </div>
 
-An atomic rule expectation MUST only use information available in the [test aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures the rule is encapsulated.
 
+### Expectations for Composite Rules ### {#expectations-composite}
 
-Expectations for Composite Rules {#expectations-composite}
---------------------------------------------------------
+All expectations of a [=composite rule=] MUST describe the logic that is used to determine a single `passed` or `failed` [=outcome=] for a [=test subject=], based on the [=findings=] of rules in its [input rules](#input-rules). The outcome for a test target is `inapplicable` when all input rules are inapplicable. 
```

Comment:
I think the sentence "All expectations of a [=composite rule=] MUST describe" should be changed to "The set of all expectations of a [=composite rules=] MUST collectively describe", because not every expectation must result in a single passed or failed but rather the collective set of one or more expectations.